src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
-body {
+$box-size: 100px;
+$margin-top: 80px;
+$gap: 1px;
+$calendar-size: calc(($box-size + $gap) * 7);
```

Comment:
you have  7 days, but 6 gaps, please calculate it here instead of subtract it in element width property